portal-web/test/functional/com/liferay/portalweb/macros/SystemSettings.macro

```diff
@@ -133,7 +133,7 @@ definition {
 	macro editTextAreaSetting {
 		Type(
 			key_label = ${settingName},
```

Comment:
I realized that the path that you created doesn’t depend on the `key_label` could you remove?